debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -304,6 +312,40 @@ public Long lastReceivedLsn() {
                 return lastReceivedLsn != null ? lastReceivedLsn.asLong() : null;
             }
 
+            @Override
+            public void startKeepAlive(ExecutorService service) {
+                if (keepAliveExecutor == null && statusUpdateIntervalMillis > 0) {
+                    LOGGER.info("starting keep alive thread");
+                    keepAliveExecutor = service;
+                    Metronome metronome = Metronome.sleeper(Duration.ofMillis(statusUpdateIntervalMillis), Clock.SYSTEM);
```

Comment:
Can you make `statusUpdateIntervalMillis` a `Duration` itself (right up to the point where the value is obtained from the config); we try to converge to the more semantic temporal types globally, so this is a good occassion to continue that effort.